code/datums/actions/xeno_action.dm

```diff
@@ -208,7 +208,7 @@
 
 /datum/action/xeno_action/activable/can_use_action(silent = FALSE, override_flags, selecting = FALSE)
 	if(selecting)
-		return ..(silent, XACT_IGNORE_COOLDOWN|XACT_IGNORE_PLASMA|XACT_USE_STAGGERED)
```

Comment:
This is for selecting abilities that are on cooldown, anything thats activable shouldn't do anything when selected.